app-android/src/main/java/com/ustadmobile/port/android/view/SiteEnterLinkFragment.kt

```diff
@@ -93,4 +126,81 @@ class SiteEnterLinkFragment : UstadBaseFragment(), SiteEnterLinkView{
         siteLink = null
         mBinding = null
     }
-}
\ No newline at end of file
+}
+
+@Composable
+private fun SiteEnterLinkScreen(
+    uiState: SiteEnterLinkUiState = SiteEnterLinkUiState(),
+    onClickNext: () -> Unit = {},
+    onClickNewLearningEnvironment: () -> Unit = {},
+    onEditTextValueChange: (String) -> Unit = {},
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally
+    )  {
+
+        Image(
+            painter = painterResource(id = R.drawable.illustration_connect),
+            contentDescription = null,
+            modifier = Modifier
+                .height(200.dp))
+
+        Text(stringResource(R.string.please_enter_the_linK))
+
+        UstadTextEditField(
+            value = "",
+            label = stringResource(id = R.string.site_link),
+            onValueChange = onEditTextValueChange,
+            error = uiState.errorMessage,
+            enabled = false,
```

Comment:
You need to add fieldsEnabled boolean to the viewModel, and then this enabled property should be controlled by the view moedl